db/migrate/20211103111313_create_notes.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateNotes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :notes do |t|
+      t.integer :planning_application_id, null: false
+      t.integer :user_id, null: false
```

Comment:
I think these two might be `t.references`

Reply:
woops, nice spot. This is what i get for writing it manually